examples/history_vehicles_replacement_for_imitation_learning.py

```diff
@@ -42,7 +40,13 @@ def main(scenarios, headless, seed):
             agent = agent_spec.build_agent()
 
             smarts.switch_ego_agent({agent_id: agent_spec.interface})
-
+            smarts.history_set_start_elapsed_time(mission.start_time)
+            modified_mission = Mission(
+                goal=agent_missions[agent_id].goal,
+                start_time=0.0,
+                start=agent_missions[agent_id].start,
+            )
```

Comment:
I would suggest using `dataclasses.replace()`.

```suggestion
            modified_mission = replace(mission, start_time=0.0)
```